integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb

```diff
@@ -23,7 +23,9 @@
     "* Initialize Neptune and create a `run`,\n",
     "* Create a `NeptuneCallback()`,\n",
     "* Log model training metrics to Neptune using `NeptuneCallback()`\n",
-    "* Log additional metadata like hyperaparameters, images, and model weights to Neptune."
+    "* Log additional metadata like hyperaparameters, images, and model weights to Neptune.\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/tf-keras-integration/e/TFK-40889/all)"
```

Comment:
@SiddhantSadangi I can't add showcase run tag to this run as I don't have the necessary rights. Please add the tag. Thanks!